### ISSUE TITLE
"LogAcousticsRuntime: Warning: No acoustic query result found for source:"

### ISSUE THREAD

**Issue by HDalsfelt** [microsoft/ProjectAcoustics#169]
Hi,

I saw there was an old (closed) issue that was that mentioned this issue. (https://github.com/microsoft/ProjectAcoustics/issues/95) We also get this log spamming when playing multiplayer, did you find out the reason why this is happening? It's affecting performance in our playtest (when we're not using shipping), would be nice to get a clue on what's happening. =)

**Comment by KyleStorck** [comment]
Yes, running multiple clients within the same editor is not a scenario that we support currently.

**Comment by HDalsfelt** [comment]
Ah, but we get it when running online multiplayer, maybe it's not related to that thread.

**Comment by KyleStorck** [comment]
Can you post what the exact multiplayer setup is like? And any Output Logs you have. I suspect it's the same issue. There is some shared client/server instance where multiple SDO plugins are live at the same time.

**Comment by HDalsfelt** [comment]
Dedicated server (linux), 12 clients, PA running only on clients. We're running with Wwise so there's no SDO(?). I guess if the bake is broken this might occur as well? It seems like it happens in certain areas on the level and also when I'm alone on the server. But running locally with the same data looks fine (and without errors).

**Comment by KyleStorck** [comment]
Oooh, interesting. We haven't seen this in Wwise yet. Also, I didn't read closely, "No acoustic query result found for source" might be a different problem. Can you provide us with as much detail as you can. Your Output Logs, the exact scenarios when these warnings happen and don't happen, maybe some videos. Are you hearing acoustic effects at all when the warning happens?

**Comment by HDalsfelt** [comment]
I hear the acoustics effect from the character as usual when the spamming is active. It occurs at certain locations on the level, for some reason I can't debug pa (probes/voxels) when running from steam but when running locally the voxels and probes look fine. It's a fine build. Here's a  log
[Game_QueryNotFInishedInTime.log](https://github.com/microsoft/ProjectAcoustics/files/11208171/Game_QueryNotFInishedInTime.log)

**Comment by HDalsfelt** [comment]
Here's a video, managed to turn on debugging. When I reach the open area it starts spamming (as you'll notice on the query-timer). When I turn on probes the log-spamming stops (but the query-thread still takes a long time). https://drive.google.com/file/d/1axrep0zKKa9Zvwxo2ykgdg0gRP6lcEnh/view?usp=sharing

**Comment by KyleStorck** [comment]
Thanks for the data. Are you using any other Project Acoustics features in your scene? Dynamic openings? Probe Volumes? Interpolation configs?

**Comment by HDalsfelt** [comment]
I've removed the dynamic openings and I use no probe volumes in this scene. I tried different interpolation configs, but no difference. We have a lot of ambient events that move along splines, when I disabled acoustics on all of those the problem disappeared. I'll check if I can find if there is any specific event that causing this problem. It's weird that it only occurs at certain spots in the level.
